services/core-api/app/__init__.py

```diff
@@ -102,7 +102,12 @@ def register_extensions(app):
     root_api_namespace.init_app(app)
 
     try:
-        jwt.init_app(app)
+
+        if test_config is None:
+            jwtv2.init_app(app)
```

Comment:
v1 is existing keycloack 
v2 is Gold SSO. 

we will have more added here once we get integration clients in gold sso 

like: 

`jwtBCMI` `jwtDataBC` `jwtGenTx` etc..